src/fromager/bootstrapper.py

```diff
@@ -227,45 +227,67 @@ def bootstrap(self, req: Requirement, req_type: RequirementType) -> None:
         In test mode, catches build exceptions, records package name, and continues.
         In normal mode, raises exceptions immediately (fail-fast).
         """
-        try:
-            self._bootstrap_impl(req, req_type)
-        except Exception as err:
-            if not self.test_mode:
-                raise
-            # Get version from cache if available
-            cached = self._resolved_requirements.get(str(req))
-            if cached:
-                _source_url, resolved_version = cached
-                version = str(resolved_version)
-            else:
-                version = None
-            self._record_test_mode_failure(req, version, err, "bootstrap")
+        logger.info(f"bootstrapping {req} as {req_type} dependency of {self.why[-1:]}")
```

Comment:
The resolver logic needs to be inside the try/except, since it is still a common source of failures.

Reply:
This is a good suggestion. Will fix it.

Reply:
Fixed it in https://github.com/python-wheel-build/fromager/commit/400544aaf7d85bf16f5c14c0690f1df47160e1d6